guides/common/modules/proc_configuring-tracer-on-a-host.adoc

```diff
@@ -1,18 +1,25 @@
-[id="enabling-tracer-on-a-host_{context}"]
-= Enabling Tracer on a host
+[id="configuring-tracer-on-a-host"]
```

Comment:
```suggestion
[id="configuring-tracer-on-a-host_{context}"]
```
If you're going to include it twice, you have to use the context.

But does it really have to be included twice?